nvtabular/ops.py

```diff
@@ -788,6 +790,46 @@ def apply_mean_std(self, gdf, stats_context, cont_names):
         return new_gdf
 
 
+class NormalizeMinMax(DFOperator):
+    """
+    Standardizing the features around 0 with a standard deviation
+    of 1 is a common technique to compare measurements that have
+    different units. This operation can be added to the workflow
+    to standardize the features.
+
+    It performs Normalization using the min max method.
+
+    Although you can directly call methods of this class to
+    transform your continuous features, it's typically used within a
+    Workflow class.
+    """
+
+    default_in = CONT
+    default_out = CONT
+
+    @property
+    def req_stats(self):
+        return [MinMax()]
+
+    @annotate("NormalizeMinMax_op", color="darkgreen", domain="nvt_python")
+    def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None):
+        cont_names = target_columns
+        if not cont_names or not stats_context["mins"]:
+            return
+        gdf = self.apply_min_max(gdf, stats_context, cont_names)
+        return gdf
+
+    def apply_min_max(self, gdf, stats_context, cont_names):
+        new_gdf = cudf.DataFrame()
+        for name in cont_names:
+            dif = stats_context["maxs"][name] - stats_context["mins"][name]
+            if dif > 0:
+                new_col = f"{name}_{self._id}"
+                new_gdf[new_col] = (gdf[name] - stats_context["means"][name]) / (dif)
```

Comment:
I don't think this is the right formula for min-max scaling - I believe it should be 
```suggestion
                new_gdf[new_col] = (gdf[name] - stats_context["mins"][name]) / (dif)
```
to actually scale to a [0, 1] range.  (I realize you implemented what Ronay asked for in the linked issue, asked a follow up question here to verify https://github.com/NVIDIA/NVTabular/issues/34#issuecomment-646850202).

If we do need the mean stat though, you will have to add Moments to the 'req_stats' return value (otherwise it won't get pulled in automatically).